superset-frontend/src/views/CRUD/data/common.ts

```diff
@@ -39,5 +39,11 @@ export const commonMenuData = {
       url: '/savedqueryview/list/',
       usesRouter: true,
     },
+    {
+      name: 'Query History',
+      label: t('Query History'),
+      url: '/superset/sqllab/search/',
```

Comment:
Is "Query History" the same as "Query Search"? I thought query search is for searching all saved queries across the whole company that is not necessarily authored by you? (Maybe that was the original intention because I see that there is a filter by user option in the current query search page---even though it doesn't really work)

If the product decision is to allow only searching one's own query history, then maybe the URL should be `/superset/sqllab/history`?

Reply:
I think the naming is a bit confusing at the moment. The goal is to eventually have "Query History" be the only listview for queries. There is a product decision to rename "Query Search" -> "Query History", there will be a user dropdown and all the other filters that currently exist on the "Query Search" page. Once we fully deprecate the old view we can rename everything to "Query History". 

I suppose since we do have the opportunity to change the URL here, `/superset/sqllab/history` does make more sense than `/superset/sqllab/search`